packages/@coorpacademy-components/src/molecule/progress-wrapper/index.js

```diff
@@ -1,44 +1,52 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {isEmpty, map} from 'lodash/fp';
+import {convert} from 'css-color-function';
+import {isEmpty, map, get} from 'lodash/fp';
 import Title from '../../atom/title';
 import ProgressBar from '../progress-bar';
 import {COLORS} from '../../variables/colors';
 import Tag from '../../atom/tag';
 import ButtonLink from '../../atom/button-link';
 import Icon from '../../atom/icon';
-import Provider from '../../atom/provider';
+import Provider, {GetSkinFromContext} from '../../atom/provider';
 // eslint-disable-next-line css-modules/no-unused-class
 import style from './style.css';
 
 const uncappedMap = map.convert({cap: false});
 
 const DetailSection = ({index, type, isLocked, downloadUrl, stars}, context) => {
   const {translate} = context;
+  const skin = GetSkinFromContext(context);
```

Comment:
remove maj ?

Reply:
React Hook "useWebContext" is called in function "getSkinFromContext" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use"